src/main/java/seedu/address/logic/parser/LabelCommandParser.java

```diff
@@ -33,6 +34,10 @@ public LabelCommand parse(String args) throws ParseException {
                 fileName = new FileName(DEFAULT_FILENAME);
             }
             index = ParserUtil.parseIndex(argMultimap.getPreamble());
+
+            if (fileName.toString().length() > 9999) {
```

Comment:
The label command stops working when the filename length is around 60 or so. Not sure about the actual length. The limit can be set as a constant field MAX_LENGTH_FILENAME.